src/components/common/MetaData/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
I thought about these `common` and `widgets` folders. Perhaps, we could unit all components from these folders in a `shared` folder?

Reply:
I agree, it seems an odd divide.